Package@swift-5.3.swift

```diff
@@ -89,16 +89,16 @@ let package = Package(
             path: "AppCenterDistribute/AppCenterDistribute",
             exclude: ["Support"],
             resources: [
-                .copy("AppCenterDistributeResources.bundle"),
+                .process("Resources/AppCenterDistribute.strings"),
```

Comment:
Why "strings", but not "bundle"?

Reply:
Based on [this](https://github.com/apple/swift-evolution/blob/master/proposals/0278-package-manager-localized-resources.md#localized-resource-discovery) guide we have to point here resources which going to copy and use based on locale.